### PR TITLE
feat: productSlider [SFUI2-1190]

### PR DESCRIPTION
**Pull request by FRSgit**
# Related issue

Closes https://vsf.atlassian.net/browse/SFUI2-1190\

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: .changeset/lucky-papayas-reflect.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@storefront-ui/vue': minor
+'@storefront-ui/react': minor
+---
+
+add ProductSlider block
```
Comment (Szymon-dziewonski) [inline .changeset/lucky-papayas-reflect.md:6]: Block is not part of our packages that we release on npm, so it should not be mentioned in changelog

File: apps/docs/components/blocks/ProductSlider.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: The ProductSlider allows for scrollable content with product cards, providing a visually appealing and interactive way to showcase products.
+hideToc: true
+
```
Comment (Szymon-dziewonski) [inline apps/docs/components/blocks/ProductSlider.md:6]:
```suggestion
```

File: .changeset/lucky-papayas-reflect.md (hunk 1 above)
Comment (FRSgit) [reply .changeset/lucky-papayas-reflect.md:6]: Yea, was wondering about that - how will we raise the version number when there are no changes to the package? 😄

Comment (Szymon-dziewonski) [reply .changeset/lucky-papayas-reflect.md:6]: What do you mean, you did changes in component from package in scrollable.vue so you should make patch change only for vue package and write `Prop typing change in SfScrollable`

Comment (FRSgit) [reply .changeset/lucky-papayas-reflect.md:6]: Yup, I did it already yesterday: https://github.com/vuestorefront/storefront-ui/pull/2819/files#diff-d78d5a6ea45c0e27ea4974822b530c55251c860f32d1f9cb9435a4d09d575cc8R2-R5

But what I mean - we need to release the vue/react packages with `2.4` next week - how do we do that?

Comment (FRSgit) [reply .changeset/lucky-papayas-reflect.md]: I've remove this changelog. But the issue of "how and when to set version at `2.4`" still persists 😄

Comment (Szymon-dziewonski) [reply .changeset/lucky-papayas-reflect.md]: With what I test today changesets bump version to 2.4 so at this point we need to only go with current new flow and at the end run yarn publish by hand and thats it

Comment (FRSgit) [reply .changeset/lucky-papayas-reflect.md]: yea, but we need to make sure that we will always have a minor bump somewhere in our tasks 😄

**Review by Szymon-dziewonski — APPROVED** [review]

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**